webpack/components/extensions/HostDetails/Tabs/PackagesTab/PackagesTab.js

```diff
@@ -348,6 +348,7 @@ export const PackagesTab = () => {
           additionalListeners={[hostId, packageStatusSelected,
             activeSortDirection, activeSortColumn]}
           fetchItems={fetchItems}
+          bookmarkController="katello_erratum_packages"
```

Comment:
Is this not `katello_host_installed_packages`?

Reply:
Will change